packages/bodiless-organisms/src/components/ListBreadcrumb.tsx

```diff
@@ -0,0 +1,176 @@
+  Separator: ComponentType<HTMLProps<HTMLSpanElement>>,
+  BreadcrumbWrapper: ComponentType<HTMLProps<HTMLUListElement>>,
+  BreadcrumbItem: ComponentType<HTMLProps<HTMLLIElement>>,
+  BreadcrumbLink: ComponentType<HTMLProps<HTMLAnchorElement> & WithNodeKeys>,
+  BreadcrumbTitle: ComponentType<HTMLProps<HTMLSpanElement> & WithNodeKeys>,
+  FinalTrail: ComponentType<HTMLProps<HTMLSpanElement>> | null,
+};
+
+type BreadcrumbProps = DesignableComponentsProps<BreadCrumbComponents>;
+
+const asBreadcrumbListItem = asBreadcrumb;
+
+const CleanBreadcrumbs = observer((props: BreadcrumbProps) => {
+  const { components, nodeCollection } = props;
+  const {
+    StartingTrail,
+    Separator,
+    BreadcrumbWrapper,
+    BreadcrumbItem,
+    BreadcrumbLink,
+    BreadcrumbTitle,
+    FinalTrail,
+  } = components;
+  const store = useBreadcrumbStore();
+  if (store === undefined) return <></>;
+  const breadcrumbTrail = store.breadcrumbTrail;
+  console.log('ListBreadcrumb');
+  console.log(store.export());
+  console.log('breadcrumbtrail');
+  console.log(breadcrumbTrail);
+  const { node } = useNode(nodeCollection);
+  const basePath = node.path;
+  const items = breadcrumbTrail.map<React.ReactNode>((item: BreadcrumbItemInterface) => {
+    // ToDo make relative path logic reusable 
+    const linkNodePath = item.getLink().nodePath.replace(`${basePath}$`, '');
+    const titleNodePath = item.getTitle().nodePath.replace(`${basePath}$`, '');
+    return (
+      <BreadcrumbItem key={item.getUUID()}>
+        <BreadcrumbLink nodeKey={linkNodePath} nodeCollection={nodeCollection}>
+          <BreadcrumbTitle nodeKey={titleNodePath} nodeCollection={nodeCollection} />
+        </BreadcrumbLink>
+      </BreadcrumbItem>
+    );
+  });
+  // join elements by separator
+  const items$1 = items.length > 1 ? items.reduce((
+    prev,
+    curr,
+    index,
+  ) => [prev, <Separator key={`item${index}`}/>, curr]) : items;
+  return (
+    <BreadcrumbWrapper>
+      { StartingTrail &&
+        <>
+          <BreadcrumbItem key="startingTrail">
+            <StartingTrail />
+          </BreadcrumbItem>
+          <Separator key="startingTrailSeparator" />
+        </>
+      }
+      {items$1}
+      { FinalTrail && !store.hasLastItem() &&
+        <>
+          <Separator key="finalTrailSeparator" />
+          <BreadcrumbItem key="finalTrail">
+            <FinalTrail />
+          </BreadcrumbItem>
+        </>
+      }
+    </BreadcrumbWrapper>
+  );
+});
+
+const designableBreadcrumb = extendDesignable((
+  {
+    StartingTrail,
+    Separator,
+    BreadcrumbWrapper,
+    BreadcrumbLink,
+    BreadcrumbItem,
+    BreadcrumbTitle,
+    FinalTrail,
+    ...rest
+// ToDo: improve types
+}: any) => rest);
+
+const DesignableCleanBreadcrumbs = designableBreadcrumb({
+  StartingTrail: null,
+  Separator: asComponent(Span),
+  BreadcrumbWrapper: asComponent(Ul),
+  BreadcrumbItem: asComponent(Li),
+  BreadcrumbLink: asComponent(A),
```

Comment:
>and use a designable compoent as the item

I like the idea with using a designable component as the item. there may be a microdata use case when we want to render link, title and meta components.

>or mimic what we do in lists

I like it less since site builders will get less flexibility. given there is a designed breadcrumb component, site builders won't be able to add design to titles (text) without recomposing the item